<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv='cache-control' content='no-cache'> 
<meta http-equiv='expires' content='0'> 
<meta http-equiv='pragma' content='no-cache'>
<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="’Kolade Gracious">
<meta name="dcterms.date" content="2023-09-18">
<meta name="description" content="Cell-based studies are important in biotechnology. But, how can a scientist avoid the usual errors and get the best result?">

<title>’Kolade Gracious - Direct Library Preparation: The Latest Innovation in Single-Cell Whole Genome Sequencing</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../favicon.io" rel="icon">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="../../theme.scss">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">’Kolade Gracious</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../data/index.html" rel="" target="">
 <span class="menu-text">Data Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../blogs/index.html" rel="" target="">
 <span class="menu-text">Medical Writing</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto tools-wide">
    <a href="https://twitter.com/DrealOtunba" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-twitter"></i></a>
    <a href="https://github.com/koladegray" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-github"></i></a>
    <a href="https://linkedin.com/gracious-ogunkolade" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-linkedin"></i></a>
    <a href="https://medium.com/gray-commits" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-medium"></i></a>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item">Direct Library Preparation: The Latest Innovation in Single-Cell Whole Genome Sequencing</li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#cell-sequencing-processes-and-studies" id="toc-cell-sequencing-processes-and-studies" class="nav-link active" data-scroll-target="#cell-sequencing-processes-and-studies"><strong>Cell Sequencing Processes and Studies</strong></a></li>
  <li><a href="#popular-methods-of-isolating-single-cells" id="toc-popular-methods-of-isolating-single-cells" class="nav-link" data-scroll-target="#popular-methods-of-isolating-single-cells"><strong>Popular Methods of Isolating Single Cells</strong></a>
  <ul class="collapse">
  <li><a href="#fluorescence-activated-cell-sorting-facs" id="toc-fluorescence-activated-cell-sorting-facs" class="nav-link" data-scroll-target="#fluorescence-activated-cell-sorting-facs">1. Fluorescence Activated Cell Sorting (FACS)</a></li>
  <li><a href="#microfluids" id="toc-microfluids" class="nav-link" data-scroll-target="#microfluids">2. Microfluids</a></li>
  <li><a href="#mechanical-micromanipulation" id="toc-mechanical-micromanipulation" class="nav-link" data-scroll-target="#mechanical-micromanipulation">3. Mechanical Micromanipulation</a></li>
  </ul></li>
  <li><a href="#direct-library-preparation-method-the-latest-innovation-in-cell-sequencing" id="toc-direct-library-preparation-method-the-latest-innovation-in-cell-sequencing" class="nav-link" data-scroll-target="#direct-library-preparation-method-the-latest-innovation-in-cell-sequencing">Direct <strong>Library Preparation Method: The Latest Innovation in Cell Sequencing</strong></a></li>
  <li><a href="#cited-works" id="toc-cited-works" class="nav-link" data-scroll-target="#cited-works">Cited Works</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Direct Library Preparation: The Latest Innovation in Single-Cell Whole Genome Sequencing</h1>
<p class="subtitle lead">Exploring a novel innovation in genome sequencing</p>
  <div class="quarto-categories">
    <div class="quarto-category">Cell</div>
    <div class="quarto-category">Sequencing</div>
    <div class="quarto-category">Biotechnology</div>
  </div>
  </div>

<div>
  <div class="description">
    Cell-based studies are important in biotechnology. But, how can a scientist avoid the usual errors and get the best result?
  </div>
</div>


<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>’Kolade Gracious </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">September 18, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<hr>
<p>Cell culture and tissue studies are becoming important in genomics and human biology. Despite the wide innovations in the field of cell studies, clinical results and experimental reviews from cell-based examinations are still largely tilted towards the conventional assumption that all cells derived from cultures and tissues are completely homogeneous.</p>
<p>How then should scientists study biological heterogeneity? Most importantly, how should they study the inherent properties of a single cell as an offshoot of the genome?</p>
<section id="cell-sequencing-processes-and-studies" class="level2">
<h2 class="anchored" data-anchor-id="cell-sequencing-processes-and-studies"><strong>Cell Sequencing Processes and Studies</strong></h2>
<p>For decades on end, cell biologists have devised multiple cell sequencing methods to navigate beyond the technical difficulties of cell examination. Scientific insight into cell cultures and tissues are basically important in cancer studies and cell other forms of cellular anomalies<span class="citation" data-cites="Wang2017"><sup><a href="#ref-Wang2017" role="doc-biblioref">5</a></sup></span>.</p>
<p>Single cell sequencing (SCS) techniques have long been considered a standard procedure in inter-cellular probing. When coupled with technological advancements in genome amplification and single cell isolation, SCS produces a valuable set in cellular studies of inherent properties at high resolution.</p>
<p>Currently, leading research centers worldwide employs Single-cell whole-genome sequencing as a routine procedure in probing intercellular genomic variations and studying single-nucleotide variations in single cells.</p>
<p>Despite the innovative results produced by these methods, studying genomic variations in with precise accuracy was still a big challenge in human medicine and cell biology. Scientists are consistently burdened with many question. The chief of which is – how exactly should we qualitatively study genomic features and cell heterogeneity?</p>
</section>
<section id="popular-methods-of-isolating-single-cells" class="level2">
<h2 class="anchored" data-anchor-id="popular-methods-of-isolating-single-cells"><strong>Popular Methods of Isolating Single Cells</strong></h2>
<p>The scientific race to isolate single cell from a whole genome started properly in 2009<span class="citation" data-cites="Yilmaz2012"><sup><a href="#ref-Yilmaz2012" role="doc-biblioref">7</a></sup></span>. Based on available evidence and the increasing need for cell-based studies in microbiology, scientists were able to analyze transcriptome complexity in individual cells using the first cell whole transcriptome sequencing protocol.</p>
<p>The scientific race to isolate single cell from a whole genome started properly in 2009. Based on available evidence and the increasing need for cell-based studies in microbiology, scientists were able to analyze transcriptome complexity in individual cells using the first cell whole transcriptome sequencing protocol.</p>
<p>Two years later, the science evolved and single-cell whole-genome sequencing was created . This breakthrough ushered in a new wave of scientific inquiries that led to the development of a single cell whole exome sequencing in 2012, and in 2013, a novel method of single cell epigenomic sequencing was developed.</p>
<p>As expected, the methods to isolating single cell from a whole genome were formed around these protocols. Currently, there are a few methods employed by different leading research institutions around the world in isolating single cells from cell culture or tissue mass. These methods, although effective to an extent, were with many shortcomings.</p>
<p>To a large extent, the general challenge associated with these old methods is in adequately controlling the quality and quantity of cell isolates extracted from tissues or cell cultures. These old methods include: <br></p>
<ol type="1">
<li><p>Fluorescence Activated Cell Sorting (FACS)</p></li>
<li><p>Microfluidics</p></li>
<li><p>Mechanical Micromanipulation</p></li>
</ol>
<p style="text-align: center;">
</p><section id="fluorescence-activated-cell-sorting-facs" class="level3">
<h3 class="anchored" data-anchor-id="fluorescence-activated-cell-sorting-facs">1. Fluorescence Activated Cell Sorting (FACS)</h3>
<p></p>
<p>With wide application in microbiology, immunology, and embryonic development, Fluorescence Activated Cell Sorting is widely used in many laboratories and genomic research facilities. A popular high point linked with FACS is the ease and efficiency with which this method can isolate multiple cells (hundreds of thousands of cells) in split timing.</p>
<p><br></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/image4.jpg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">FACS</figcaption>
</figure>
</div>
<p><br></p>
<p>The protocol of operation of cell isolation in FACS is centered on</p>
<ul>
<li><p>Cell size</p></li>
<li><p>Cell Fluorescence Properties</p></li>
<li><p>Cell Granularity</p></li>
</ul>
<p>This method is generally considered efficient and fast as it creates a unique system of cell sorting. When in use, FACS can easily isolate designated cells from a tissue sample or genome by exploring the ability of these cells to fluorescence when pre-labeled.</p>
<p>This unique property of the FACS cell-sorting protocol explains its popularity in the study f single viral particles in a poll or assemblage of mixed viral genome. When compared with recently developed protocols, such as the Direct Library Preparation Plus, the Fluorescence Activated Cell Sorting method is limited in application.</p>
<p>In this method, it is compulsory that a bulk of the cell assemblage be prepared as sorting material and samples must be prepared in solution. Earlier observation reports also holds that the fluorescent dye used in cell labeling can damage the viability of cells or influence the inherent properties of the single cells<span class="citation" data-cites="Hodne2015"><sup><a href="#ref-Hodne2015" role="doc-biblioref">1</a></sup></span>.</p>
</section>
<section id="microfluids" class="level3">
<h3 class="anchored" data-anchor-id="microfluids">2. Microfluids</h3>
<p>The shortcomings with FACS, especially the need for an abundant cell assemblage and risk of losing cell viability, led to the development of Microfluidics. This method employs a protocol setting completely different in mechanism to the FACS.</p>
<p>Microfluidics is widely used in</p>
<ul>
<li><p>Microbiology</p></li>
<li><p>Embryonic development studies</p></li>
<li><p>Neurobiology</p></li>
</ul>
<p>Basically, the Microfluidics setup<span class="citation" data-cites="Yilmaz2012"><sup><a href="#ref-Yilmaz2012" role="doc-biblioref">7</a></sup></span> uses a highly integrated system to achieve single cell culture and sequencing by manipulating small volumes of a few hundreds of micro-liters of fluids. As the nomenclature suggests, Microfluidics needs only a few volume of fluids in cell-based studies<span class="citation" data-cites="liu2020"><sup><a href="#ref-liu2020" role="doc-biblioref">3</a></sup></span>.</p>
<p><br></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/image3.jpg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Microfluidics Setup</figcaption>
</figure>
</div>
<p><br></p>
<p>Research institutes from around the world are currently experimenting with the feasibility of employing Microfluidics in human medicine especially in the areas of epigenomics sequencing and single cell whole-genome studies.</p>
<p>In the laboratories and microbiological studies, Microfluidics can effectively separate biological nanoparticles from tissues samples, cell cultures and cell assemblage. In a 2016 publication of <em>Nature,</em> the future of Microfluidics was examined based on coverage and advantage in cell sequencing<span class="citation" data-cites="whitesides2006"><sup><a href="#ref-whitesides2006" role="doc-biblioref">6</a></sup></span>.</p>
<p>Reports from this review suggest that Microfluidics produces accurate results with a unique level of sensitivity in almost all research applications as reported. This method can also finely resolve nanoliter-to-picoliter volumes of samples in research studies that need timely analysis and effective outcomes in low volume fluids.</p>
</section>
<section id="mechanical-micromanipulation" class="level3">
<h3 class="anchored" data-anchor-id="mechanical-micromanipulation">3. Mechanical Micromanipulation</h3>
<p>As an old and classic method in cell isolation and single-cell based studies, Mechanical Micromanipulation surely deserves an honorable mention. This manual method is cost-efficient and can only need cheap instrumentation especially when results are not needed to be extremely accurate.</p>
<p>This explains the continued use of Mechanical Micromanipulation in many Microbiology research laboratories around the world, despite the popularity of new methods. The Mechanical Micromanipulation setup only requires the mechanical suctioning of a single cell from a cell assemblage using a capillary pipette, microscope and other supporting instruments<span class="citation" data-cites="ishii2010"><sup><a href="#ref-ishii2010" role="doc-biblioref">2</a></sup></span>.</p>
<p><br></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/image2.jpg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Mechanical Manipulation setup</figcaption>
</figure>
</div>
<p><br></p>
<p>Cell isolation is not automatic and as such requires personnel guidance in the visual inspection of single-cell color features and morphological characteristics. The shortcoming of Mechanical Micromanipulation is centered on its manual processes.</p>
<p>Ranging from personnel faults, to mechanical shearing during manipulation and the possibility of cell damage, the shortcoming of Mechanical Micromanipulation appears to be unlimited in perspective.</p>
<p>Although, the microscope provides a means of exact cell identification in a tissue sample, an unskilled personnel might misidentify the target cell and the whole process is time-consuming. These shortcoming are solved in Microfluidics and the Fluorescence Activated Cell Sorting method</p>
</section>
</section>
<section id="direct-library-preparation-method-the-latest-innovation-in-cell-sequencing" class="level2">
<h2 class="anchored" data-anchor-id="direct-library-preparation-method-the-latest-innovation-in-cell-sequencing">Direct <strong>Library Preparation Method: The Latest Innovation in Cell Sequencing</strong></h2>
<p>The Direct Library Preparation method of cell sequencing was developed as a scalable single-cell whole-genome sequencing platform. Compared to other methods before this innovation, the Direct Library Preparation instrumentation is perfectly designed for accuracy, high resolution and unparalleled efficiency in cell isolation experiments and research findings.</p>
<p>Recently published biotech reviews on this subject matter have positioned DLP as more than just an innovation in cell isolation studies. By large, it ushers in an era where microbiology and human medicine can optimally benefit from technology.</p>
<p><br></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/image5.jpg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">DLP</figcaption>
</figure>
</div>
<p><br></p>
<p>The instrumentation boasts of open source computational methods, image-based object recognition and commodity instrument. This innovation is widely used in many research finding for the identification of clonal populations and their corresponding genomic features.</p>
<p>Direct Library Preparation has shown useful prospects in</p>
<ul>
<li><p>Tumor studies</p></li>
<li><p>Microbiology</p></li>
<li><p>Neurology</p></li>
<li><p>Immunology</p></li>
<li><p>Embryonic development studies</p></li>
</ul>
<p>By extension, there are theoretical evidences supporting the use of this novel method in genome heterogeneity studies, mutational processes and findings on clonal evolution in healthy and cancer tissues. Early reviews and studies on the Direct Library Preparation platform showed that it can capture a high-resolution microscopy images of cells. These images are captured as the cells navigates a transparent nozzle as they settle in wells.</p>
<p>Image output allows for exact cell identification and separation. All single cell captures are lysed processes to produce unique sequencing inserts. Standard illumina protocols and allows the researcher to pool and sequence indexed libraries at the desired coverage depth.</p>
<p>The wide application of Direct Library Preparation in the research might also be linked with the innovative advantage of the development and inclusion of an open-source, cloud compatible software infrastructure. This inclusion allows the bulk storage of data and metadata produced in all sequencing and imaging processes. Based on a researcher’s preference, the result generated from this storage can be loaded into selected data visualization and assessment platform for further studies, data exploration and quality control.</p>
<p>An early report on the application of Direct Library Preparation was published by a January 2020 issue of <em>Nature Methods</em>. The authors in this study use Direct Library Preparation to generate a resource of 51,926 single-cell genomes and matched cell images from diverse cell types including cell lines, xenograft and diagnostic samples with limited material. The resource bulk generated allows the study of variations in mitotic mis-segregation rates across tissues types and genotypes. As expected, the analysis of genomic matches obtained and images captured aided in establishing a correlation between cellular morphology and genome ploidy states<span class="citation" data-cites="todorovic2020"><sup><a href="#ref-todorovic2020" role="doc-biblioref">4</a></sup></span>.</p>
<p>This single review on DLP argued a case for its integration into medical sciences especially in tumor and embryonic development studies. This study, as reported, also documented the calculation of single-nucleotide resolution clonal genotypes and inferences of clonal phylogenies using aggregation of cells sharing copy number profiles. As a last result in this study, the authors also reported clone-specific chromosomal aneuploidy in polyclonal populations as defined by joint analysis of the listed features. Speculations on the readiness of DLP’s use in human medicine are currently on debate.</p>
<p>However, there are enough evidence to argue for its usefulness in tumor studies. By allowing the measurement of clonal replication states and rare aneuploidy patterns of single cells, DLP is evidently suitable for embryonic development and tumor tracking studies.</p>
<section id="comparison-benefits-and-features" class="level4">
<h4 class="anchored" data-anchor-id="comparison-benefits-and-features"><strong>Comparison, Benefits and Features</strong></h4>
<p>Microfluidics, Mechanical Micromanipulation and Fluorescence Activated Cell Sorting are methods developed early in cell isolation and single cell studies. However, the shortcoming of these methods have called for their replacement in many research studies and laboratory institutes.</p>
<p>The biggest challenge posed by these methods is that samples must be prepared in suspension and thus, the spatial location of the target cells in the tissues or cell cultures can be lost easily. Surprisingly, the Direct Library Preparation protocol is affordable and can be widely used on scalable methods in human and cellular studies.</p>
<p>This method is also widely considered a low bias method that can allow the spontaneous and in-depth analysis of variety of samples. In essence, this method can be effectively employed in genomic matching of tissues, cell assemblage and cell cultures.</p>
</section>
<section id="how-direct-library-preparation-solves-the-problems-in-cell-sequencing" class="level4">
<h4 class="anchored" data-anchor-id="how-direct-library-preparation-solves-the-problems-in-cell-sequencing"><strong>How Direct Library Preparation Solves the Problems in Cell Sequencing</strong></h4>
<p>Unlike other methods, the Direct Library Preparation method also allows the determination of copy number variations (CNV), single nucleotide polymorphisms (SNPs), phylogenic reconstructions and cluster analyses.</p>
<p>The arrays of benefits of using the DLP in research and experimental findings are limitless and include:</p>
<ul>
<li><p>Low reagent cost</p></li>
<li><p>High quality of singe cell whole genome libraries</p></li>
<li><p>Lineage reconstructions</p></li>
<li><p>Use in the analysis of any cell and nuclei from 5 to 80 micrometer.</p></li>
</ul>
<p>To a large extent, DLP solved the myriads of shortcomings associated with the early methods employed in cell sequencing researches. Using the Direct Library Preparation (DLP) method in single-cell whole-genome sequencing of clinical sample is currently considered one of the best approach for cell isolation in modern biotechnology.</p>
<p>This is coming at a time when Direct Library Preparation in human medicine is increasingly becoming a topic of global scientific interest<span class="citation" data-cites="zhang2020a"><sup><a href="#ref-zhang2020a" role="doc-biblioref">8</a></sup></span>. With time, more adoption of Direct Library Preparation method is expected to be recorded in many research facilities around the world.</p>
</section>
</section>
<section id="cited-works" class="level2">




</section>

<a onclick="window.scrollTo(0, 0); return false;" role="button" id="quarto-back-to-top"><i class="bi bi-arrow-up"></i> Back to top</a><div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">Cited Works</h2><div id="refs" class="references csl-bib-body" role="list">
<div id="ref-Hodne2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Hodne K, Weltzien FA. Single-Cell Isolation and Gene Analysis: Pitfalls and Possibilities. <em>International Journal of Molecular Sciences</em>. 2015;16(11):26832-26849. doi:<a href="https://doi.org/10.3390/ijms161125996">10.3390/ijms161125996</a></div>
</div>
<div id="ref-ishii2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Ishii S, Tago K, Senoo K. Single-cell analysis and isolation for microbiology and biotechnology: methods and applications. <em>Applied Microbiology and Biotechnology</em>. 2010;86(5):1281-1292. doi:<a href="https://doi.org/10.1007/s00253-010-2524-4">10.1007/s00253-010-2524-4</a></div>
</div>
<div id="ref-liu2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Liu W, He H, Zheng SY. Microfluidics in Single-Cell Virology: Technologies and Applications. <em>Trends in Biotechnology</em>. 2020;38(12):1360-1372. doi:<a href="https://doi.org/10.1016/j.tibtech.2020.04.010">10.1016/j.tibtech.2020.04.010</a></div>
</div>
<div id="ref-todorovic2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Todorovic V. Amplification-free single-cell whole-genome sequencing gets a makeover. <em>Nature Methods</em>. 2020;17(1):27-27. doi:<a href="https://doi.org/10.1038/s41592-019-0722-2">10.1038/s41592-019-0722-2</a></div>
</div>
<div id="ref-Wang2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Wang J, Song Y. Single cell sequencing: a distinct new field. <em>Clinical and Translational Medicine</em>. 2017;6(1). doi:<a href="https://doi.org/10.1186/s40169-017-0139-4">10.1186/s40169-017-0139-4</a></div>
</div>
<div id="ref-whitesides2006" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Whitesides GM. The origins and the future of microfluidics. <em>Nature</em>. 2006;442(7101):368-373. doi:<a href="https://doi.org/10.1038/nature05058">10.1038/nature05058</a></div>
</div>
<div id="ref-Yilmaz2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Yilmaz S, Singh AK. Single cell genome sequencing. <em>Current Opinion in Biotechnology</em>. 2012;23(3):437-443. doi:<a href="https://doi.org/10.1016/j.copbio.2011.11.018">10.1016/j.copbio.2011.11.018</a></div>
</div>
<div id="ref-zhang2020a" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Zhang J, Wang W, Huang J, Wang X, Zeng Y. How far is single<span>-</span>cell sequencing from clinical application? <em>Clinical and Translational Medicine</em>. 2020;10(3). doi:<a href="https://doi.org/10.1002/ctm2.117">10.1002/ctm2.117</a></div>
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'alternate';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">Copyright 2023. ’Kolade Gracious</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>



</body></html>
